File: net46/Misc/BarRepository.vb
' --------------------------------------------------------------------------------------------------------------------
' <copyright file="BarRepository.vb" company="Inspire IT Ltd">
'   Copyright © Inspire IT Ltd. All Rights Reserved.
' </copyright>
' --------------------------------------------------------------------------------------------------------------------

Imports RyanPenfold.NHibernateRepository
Imports RyanPenfold.TestProject.Model
Imports RyanPenfold.TestProject.Model.RepositoryInterfaces

''' <summary>
''' Provides data access functionality relating to <see cref="Bar" /> objects.
''' </summary>
Public Class BarRepository
	Inherits BaseRepository(Of Bar)
	Implements IBarRepository

End Class

File: net45/Misc/BarRepository.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="BarRepository.cs" company="Inspire IT Ltd">
//   Copyright © Inspire IT Ltd. All Rights Reserved.
// </copyright>
// --------------------------------------------------------------------------------------------------------------------

namespace RyanPenfold.TestProject.Repository
{
    using Model;
    using Model.RepositoryInterfaces;

    using NHibernateRepository;

    /// <summary>
    /// Provides data access functionality relating to <see cref="Bar" /> objects.
    /// </summary>
    public class BarRepository : BaseRepository<Bar>, IBarRepository
    {
    }
}